src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -502,6 +502,18 @@ namespace ReportSizingManager {
                     }
                 }
             } else {
+
+                int tempZoneNum = CurZoneEqNum;
+                bool FinalZoneSizingNotAllocated = false;
+                if (!allocated(DataSizing::FinalZoneSizing)) {
+                    DataSizing::FinalZoneSizing.allocate(1);
+                    DataSizing::ZoneSizingData &finalZoneSizing = DataSizing::FinalZoneSizing(1);
```

Comment:
The finalZoneSizing reference initialization here at line 510 is not needed and redundant since this is done at line 515. This line could be deleted but doesn't hurt anything.